submissions/api.py

```diff
@@ -764,6 +770,14 @@ def set_score(submission_uuid, points_earned, points_possible):
     try:
         score_model = score.save()
         _log_score(score_model)
+        if score_creator is not None:
```

Comment:
What's the relationship between annotated and non-annotated scores? Is it possible for a score to exist, but not be annotated? Using `score_creator` in this conditional seems to imply that any score that has been created will be annotated.

If that's not the case, could we rename this variable to `annotation_creator`, or something else that more clearly illustrates the relationship between the annotator, the original score, and the annotated score?